### PR TITLE
move `nvm` from Incubating to At-Large

### PR DESCRIPTION
**Pull request by ljharb**
With the charter approval in #750, nvm is ready to graduate incubation!

Closes https://github.com/openjs-foundation/project-status/issues/2.

### CODE REVIEW

**Review by mikesamuel — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -59,15 +60,14 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
-| <img alt="logo of project Sizzle" src="https://github.com/openjs-foundation/artwork/raw/main/projects/sizzle/sizzle-icon-color.svg" width="30" /> | [Sizzle](https://sizzlejs.com/)                   |    | [Contributing Guide](https://github.com/jquery/sizzle/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project Sizzle" src="https://github.com/openjs-foundation/artwork/raw/main/projects/sizzle/sizzle-icon-color.svg" width="30" /> | [Sizzle](https://sizzlejs.com/)                   |    | [Contributing Guide](https://github.com/jquery/sizzle/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md)
```
Comment (joesepi) [inline README.md:63]: It looks like for some reason (linter?) the last bar (`|`) is removed from this table.

File: README.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ The following projects are official OpenJS Foundation projects. If you are inter
...
 | <img alt="logo of project webhint" src="https://github.com/openjs-foundation/artwork/raw/main/projects/webhint/webhint-icon-color.svg" width="30" /> | [webhint](https://webhint.io/)          |    | [Contributing Guide](https://github.com/webhintio/hint/blob/HEAD/.github/CONTRIBUTING.md) | [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md) |
+| <img alt="logo of project nvm" src="https://github.com/openjs-foundation/artwork/raw/HEAD/projects/nvm/nvm-logo-grayscale.svg" width="30" />  | [nvm](https://nvm.sh/) | [Charter](https://github.com/nvm-sh/nvm/blob/HEAD/PROJECT_CHARTER.md) | [Contributing Guide](https://github.com/nvm-sh/nvm/blob/HEAD/CONTRIBUTING.md) | [Code of Conduct](https://github.com/nvm-sh/nvm/blob/HEAD/CODE_OF_CONDUCT.md) |
```
Comment (joesepi) [inline README.md:53]: This should be alphabetical

**Review by joesepi — APPROVED** [review]
LGTM besides my couple nits. Thanks! 🎉

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by tobie — APPROVED** [review]

**Pull request merged**